src/app/api/participant.py

```diff
@@ -0,0 +1,39 @@
+from app.RatingCalculator import RatingCalculator
+from app.api.base import BaseAuthResource
+from app.participant.models import ParticipantModel, create_participant, delete_participant, update_participant
+from flask.ext.restful import marshal, fields
+
+__author__ = 'Alex'
+
+participant_template = {
+    'league_id': fields.String,
+    'participant_id': fields.String,
+    'name': fields.String,
+    'rating': fields.String
+}
+
+
+class ParticipantListAPI(BaseAuthResource):
+    def get(self, league_id):
+        participants = ParticipantModel.query(league_id=league_id).fetch()
+        return {'data': [marshal(1, participant_template) for 1 in participants]}
+
+    def post(self):
+        new_participant = create_participant(self.user.user_id, self.args.get('league_id'),
+                                             self.args.get('name'), rating=self.args.get('rating'))
+        return {'data': marshal(new_participant, participant_template)}
+
+
+class ParticipantAPI(BaseAuthResource):
+    def get(self, participant_id):
+        participant = ParticipantModel.build_key(participant_id=participant_id).get()
+        return {'data': marshal(participant, participant_template)}
+
+    def put(self, participant_id, opponent_id, winner=None):
+        r = RatingCalculator(participant_id, opponent_id, winner=winner).process()
```

Comment:
``` process() ``` returns two values and you need to catch them both for this code to run. Right now you only catch one. Returning two values is not the same as returning an array.
``` p, o = RatingCalculator(participant_id, opponent_id, winner=winner).process() ``` is how you assign multiple return values to different variables at once.